### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_Buttons_DevTools: The "with icon only" button has no name property assigned

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#179]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG4.1.2;#AINotInScope;#DevTools;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to Button and tab to "with icon only" button , open developer tool (ctrl+shift+i) check if name is provided to it.

### Actual:
The "with icon only" button has no name property assigned.

### Expected:
The "with icon only" button should have name property assigned like for eg. here "with icon only" can be given as a name.

### User Impact:
Screen reader users will not get the proper information about the button.

**Comment by fsteffi** [comment]
**GithubTags:**#Closed;
